talk/owt/BUILD.gn

```diff
@@ -203,6 +203,7 @@ static_library("owt_sdk_base") {
       "sdk/base/win/videorendererwin.h",
       "sdk/base/windowcapturer.cc",
     ]
+    public_deps += [ "//third_party/webrtc/modules/audio_device:audio_device_module_from_input_and_output" ]
```

Comment:
Did you forget to format this file? You could run `gn format <path>` to format GN files.

Reply:
gn format will give the same result for this line. However I can update the whole file with gn format applied.

Reply:
done with the update of this file.